### ISSUE TITLE
Security vulnerabilities in v6.14.17

### ISSUE THREAD

**Issue by dShringi** [npm/cli#5591]
Using npm v6.14.17, twistlock reports the following vulnerabilities will these be fixed in v6?

**PRISMA-2022-0039:** Impacted versions: minimatch <3.0.5
minimatch package versions before 3.0.5 are vulnerable to Regular Expression Denial of Service (ReDoS). It's possible to cause a denial of service when calling function braceExpand (The regex /\\{.*\\}/ is vulnerable and can be exploited).

**CVE-2021-43616:** The npm ci command in npm 7.x and 8.x through 8.1.3 proceeds with an installation even if dependency information in package-lock.json differs from package.json. This behavior is inconsistent with the documentation, and makes it easier for attackers to install malware that was supposed to have been blocked by an exact version match requirement in package-lock.json.

**Comment by lukekarrys** [comment]
6.14.18 was released that includes `minimatch@3.1.2`. The other one is not a priority to get landed in npm 6 and it is recommended to update to npm 9.
